extras/repos/dave_sim.repos

```diff
@@ -1,65 +1,37 @@
 repositories:
-  dave/dave:
-    type: git
-    url: https://github.com/Field-Robotics-Lab/dave.git
-    version: master
-
-  dave/acoustic_msgs:
+  acoustic_msgs:
     type: git
     url: https://github.com/apl-ocean-engineering/acoustic_msgs.git
     version: main
-
-  dave/frl_msgs:
+  dave:
     type: git
-    url: https://github.com/Field-Robotics-Lab/frl_msgs.git
-    version: master
-
-  dave/gtri_based_sonar:
+    url: git@github.com:Field-Robotics-Lab/dave.git
+    version: bsb/update_2_fix_dockerfile
+  ds_msgs:
     type: git
-    url: https://github.com/Field-Robotics-Lab/gtri_based_sonar.git
+    url: https://bitbucket.org/whoidsl/ds_msgs.git
```

Comment:
This `url` might need to be `Field-Robotics-Lab`
I had an error about `ds_sensor_msgs` when I compiled using the BitBucket repo.